google/cloud/spanner/samples/samples.cc

```diff
@@ -1747,7 +1747,7 @@ void RestoreDatabaseWithEncryptionKeyCommand(std::vector<std::string> argv) {
 // [START spanner_restore_backup_with_MR_CMEK]
 void RestoreDatabaseWithMRCMEK(
     google::cloud::spanner_admin::DatabaseAdminClient client,
-    BackupIdentifier src, std::string const& database_id,
+    BackupIdentifier const& src, std::string const& database_id,
```

Comment:
odd that we are making a spanner change in this PR, but it looks like a good change